app/Services/lolapi.js

```diff
@@ -3,6 +3,13 @@ var app = angular.module("LeagueViewer");
 angular.module("LeagueViewer").factory("lolapi", function($http) {
   var baseUrl = "http://localhost:3000/";
 
+  var getSummoner1 = function(summonerName) {
```

Comment:
Let's come up with a better name to distinguish one vs. the other (getSummoner vs getSummoner1)